src/App.css

```diff
@@ -28,38 +39,169 @@
 
 .nav-bar a {
   text-decoration: none;
-  font-size: 1.5rem;
-  color: beige;
-  padding-left: 2rem;
+  font-size: 0.813rem;
+  color: #121212;
+  letter-spacing: 1.9px;
+}
+
+.nav-bar a.active {
+  color: whitesmoke;
+  background-color: #0290ff;
+  padding: 10px;
+  font-size: 15px;
 }
 
 .logo {
-  color: beige;
-  font-size: 2rem;
+  color: #0290ff;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1px;
+  padding-left: 6.25rem;
+}
+
+.profile-icon {
+  width: 2.813rem;
+  margin-right: 6.25rem;
+  height: 2.813rem;
+  padding: 0.7rem;
+  border-radius: 50%;
+  cursor: pointer;
+  border: 1px solid #e8e8e8;
+}
+
+.user-icon {
+  width: 1.063rem;
+  height: 1.063rem;
+  color: #0290ff;
+}
+
+/* ------------------------ Book List Style -------------------------- */
+
+.main-wrapper {
+  display: grid;
+  grid-template-columns: 100%;
+  margin: 6.313rem 6.25rem 0;
 }
 
 .book-wrapper {
-  margin: 5rem 3rem 1rem 2rem;
-  border-radius: 5px;
-  max-width: 40rem;
-  background-color: rgb(255, 255, 251);
-  padding: 1rem;
-  border-bottom: 1px solid;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 74rem;
+  height: 10.625rem;
+  margin-top: 0.938rem;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
+.btn {
+  width: 4.563rem;
+  height: 1.188rem;
+  margin: 0.588rem 0.4rem 0.125rem 0;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+  border: none;
+  background: none;
+}
+
+.left-line {
+  width: 0.125rem;
+  height: 1.5rem;
+  margin: 0.75rem 1.063rem 0 0.938rem;
+  border: solid 1px #e8e8e8;
+}
+
+.progress-bar {
+  width: 3.25rem;
+  height: 3.25rem;
+  margin-right: 3rem;
+}
+
+.left-line2 {
+  width: 0.125rem;
+  height: 4.375rem;
+  margin: 1.125rem 3.688rem 0 4.938rem;
+  border: solid 1px #e8e8e8;
+}
+
+.wrapper {
+  display: flex;
+  align-items: center;
+}
+
+.current-chap {
+  margin: 0.063rem 3.625rem 0 3.688rem;
+  opacity: 0.5;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  color: #121212;
+}
+
+.update-btn {
+  width: 11.5rem;
+  height: 2.063rem;
+  margin: 0.75rem 0 0.563rem 3.688rem;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  color: #fff;
+  border: none;
+}
+
+.line {
+  width: 74rem;
+  height: 0.125rem;
+  margin: 2.5rem 0.063rem 1.813rem 0;
+  border: solid 1px #e8e8e8;
+}
+
+/* ------------------------- Form Styling ------------------------ */
+
+form {
+  margin: 0 2rem 3rem 6.25rem;
 }
```

Comment:
- [ ] I noticed that some style declarations were used regularly for different elements. Instead of repeating the same style, you can put the style in a class and use the class selector for all the elements that need that style. You could also use the CSS variables for colours that you constantly use. 😎 Kindly make this change for the different style declarations like:
- `display: flex;`
- `color: #0290ff;`
- `color: #121212;`
- `border: solid 1px #e8e8e8;`
- `font-family: 'Roboto Slab', serif;`

Hint: You could combine class selectors and CSS variables to make your CSS stylesheet DRY. You could also consider using the DRY selector rule for JSX you used React inline styling for. To learn more about CSS variables, kindly read this [article](https://www.w3schools.com/css/css3_variables.asp). To learn more about not repeating the same style declaration using the class selectors, kindly read this [article](https://speckyboy.com/good-bad-css-practices/?). 💻 👍